helm-charts/common/llm-uservice/values.yaml

```diff
@@ -24,7 +24,7 @@ podSecurityContext: {}
   # fsGroup: 2000
 
 securityContext:
-  readOnlyRootFilesystem: true
+  readOnlyRootFilesystem: false
```

Comment:
Any reason the flag is set back to false?

Reply:
because latest llm-uservice need to install python module upon start time

Reply:
OK, https://github.com/opea-project/GenAIComps/pull/274 made that change.